postman/AskDarcel%20API.postman_collection.json

```diff
@@ -1,8 +1,8 @@
 {
 	"info": {
-		"_postman_id": "1f00be4c-4f11-4604-be5c-9548c7b31ab4",
+		"_postman_id": "82560b98-a655-4d4e-9b11-b784f4de9583",
 		"name": "AskDarcel API",
-		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+		"schema": "https://schema.getpostman.com/json/collection/v2.0.0/collection.json"
```

Comment:
I think you accidentally reverted the version of the Postman JSON file schema, which is causing a large number of unrelated changed lines in the diff and consequently makes it more difficult to review the changes. Could you make sure your Postman is up to date and regenerate this file without all of the deleted lines?